### PR TITLE
Fix progress bar visualization in Safari

### LINKED ISSUE #3988: Progress bar visualization doesn't render in Safari
This is on OS X El Capitan, Safari version 10.0.2 (11602.3.12.0.1), Metabase v0.22.0-snapshot (5a5981d). Happens both on question detail as well as dashboard cards.

screen shot 2016-12-20 at 11 33 48 am

### PR DESCRIPTION
**Pull request by tlrobinson**
Resolves #3988

### CODE REVIEW

File: frontend/src/metabase/visualizations/Progress.jsx [hunk 1]
```diff
@@ -127,7 +132,7 @@ export default class Progress extends Component {
                             }}
                         />
                     </div>
-                    <div className="relative" style={{
+                    <div ref="bar" className="relative" style={{
                         backgroundColor: restColor,
                         borderRadius: BORDER_RADIUS,
                         height: "25%",
```
Comment (kdoh) [inline frontend/src/metabase/visualizations/Progress.jsx]: If we have to set the height via refs I'd prefer not to set a height in the style object and just do it in one place.

Comment (tlrobinson) [inline frontend/src/metabase/visualizations/Progress.jsx]: Good point, done.

**Review by kdoh — APPROVED** [review]

**Comment by tlrobinson** [comment]
Did we want this in 0.22.0, 0.22.1 or 0.23.0?

**Comment by mazameli** [comment]
I think there was talk of 0.22.1?

**Comment by salsakran** [comment]
I think at this point, let's put all further bug fixes in 0.22.1

**Pull request merged**